architecture-baseline.json

```diff
@@ -19,5 +19,12 @@
         "rule": "BridgeMethodsInterfaceRule",
         "ruleset": "Spryker",
         "priority": "2"
+    },
+    {
+        "fileName": "src/Spryker/Zed/EventBehavior/EventBehaviorConfig.php",
+        "description": "Getter `Spryker\\Zed\\EventBehavior\\EventBehaviorConfig::getEventBehaviorTriggeringStatus()` must not return bool.",
```

Comment:
False positive? Or what should be the method name?

Reply:
Judging by the const name used in this config it might be `isTtriggeringActivated`.

Reply:
We could accept Status though maybe as exception for now